### PR TITLE
Add main entry to package.json

### PR DESCRIPTION
**Pull request by wytrych**

### CODE REVIEW

**Comment by wytrych** [comment]
@incuna/js Please review

**Comment by wytrych** [comment]
@incuna/js bump

**Comment by maxpeterson** [comment]
Looks good but needs master merging in

File: changelog.md [hunk 1]
```diff
@@ -4,6 +4,7 @@ This project uses Semantic Versioning (2.0).
 
 # Upcoming
 * Make sure npm package only installs web folder
+* Expose es5 files and make deafault entrypoint to them
```
Comment (Minglee01) [inline changelog.md]: deafault  should be `default`

Comment (wytrych) [inline changelog.md]: Will fix in next PR thanks

**Pull request merged**